packages/shared-state-reference-state/Makefile

```diff
@@ -0,0 +1,38 @@
+#
+# Copyright (C) 2019 Gioacchino Mazzurco <gio@altermundi.net>
+# Copyright (C) 2023 Javier Jorge <jjorge@inti.gob.ar>
+#
+# This is free software, licensed under the GNU Affero General Public License v3.
+#
+
+include $(TOPDIR)/rules.mk
+
+GIT_COMMIT_DATE:=$(shell git log -n 1 --pretty=%ad --date=short . )
+GIT_COMMIT_TSTAMP:=$(shell git log -n 1 --pretty=%at . )
+
+PKG_NAME:=shared-state-wifi_links_info
```

Comment:
Is the package name still appropriate?

Reply:
ups copy paste, this make file is no longer nedded